src/test/cypress/support/pageobjects/exam/ExamStartEndPage.ts

```diff
@@ -47,7 +47,7 @@ export class ExamStartEndPage {
     }
 
     pressShowSummary() {
-        cy.intercept(GET, COURSE_BASE + '*/exams/*/student-exams/*/summary').as('examSummaryDownload');
+        cy.intercept(GET, COURSE_BASE + '/*/exams/*/student-exams/*/summary').as('examSummaryDownload');
```

Comment:
Looks good - but perhaps you can make it consistent with the changes in #8161 here

```suggestion
        cy.intercept(GET, `${COURSE_BASE}/*/exams/*/student-exams/*/summary`).as('examSummaryDownload');
```

Reply:
If you insist :P